en/getting_started/basic_flight_modes.md

```diff
@@ -2,185 +2,178 @@
 
```

Comment:
@tubeme I think this should be called *Flight Modes Introduction*, "Flight Mode Concepts*, *Flight Modes Overview*. It isn't "Basic Flight Modes" because that implies the topic "Just the basic flight modes listed here".

Reply:
Thanks